app/assets/stylesheets/tabs.scss

```diff
@@ -0,0 +1,12 @@
+.content-box {
+  border: 1px solid red;
+}
+
+.section-box {
+  border: 1px solid orange;
```

Comment:
Color `orange` should be written in hexadecimal form as `#ffa500`<br>Color literals like `orange` should only be used in variable declarations; they should be referred to via variable everywhere else.